packages/agoric-cli/test/snapshots/test-inter-cli.js.md

```diff
@@ -94,6 +99,23 @@ Generated by [AVA](https://avajs.dev).
     ␊
     Options:␊
       --from <address>  wallet address literal or name␊
+      --all             show exited bids as well␊
+      -h, --help        display help for command`
+
+## Usage: inter bid cancel
+
+> Command usage:
+
+    `Usage: agops inter bid cancel [options] <id>␊
+    ␊
+    Try to exit a bid offer␊
+    ␊
+    Arguments:␊
+      id                offer id (as from bid list)␊
+    ␊
+    Options:␊
+      --from <address>  wallet address literal or name␊
+      --generate-only   print wallet action only␊
       -h, --help        display help for command`
 
 ## Usage: inter reserve add
```

Comment:
I noticed we already have a `commands/reserve.js`. There's no particular reason to treat this as part of a bidding CLI. I better move it.

Reply:
a separate PR would be better, in fact.